Gemfile

```diff
@@ -23,6 +23,17 @@ group :development do
   gem 'capistrano-bundler', require: false
   gem 'capistrano-rails', require: false
   gem 'capistrano-passenger', require: false
-  gem 'pry-byebug'
   gem 'timecop', '~> 0.9.1'
 end
+
+group :development, :test do
+  gem 'factory_bot_rails'
+  gem 'pry-byebug'
+end
+
+group :test do
```

Comment:
You're going to need the gem `puma`. See [here](https://github.com/umts/screaming-dinosaur/pull/106)

Reply:
Hooray! thanks